app/views/users/show.html.erb

```diff
@@ -20,6 +20,12 @@
         <% end %>
     <% end%>
 
+    <% if can? :destroy, @post %>
+          <%= form_with url: user_post_path(@user.id, post.id), :method => :delete do |f| %>
+            <%= f.submit 'Delete post', class: 'btn-danger btn-sm mx-2' %>
+          <% end %>
+        <% end %>
+
```

Comment:
- [ ] Kindly solve this error 

![image](https://user-images.githubusercontent.com/65068771/193325690-a22ef2d2-672f-4e71-9387-d6e4bbda5a79.png)